R/centrality.R

```diff
@@ -1104,7 +1104,8 @@ diversity <- diversity_impl
 #'   random surfer model, an edge with a larger weight is more likely to be
 #'   selected by the surfer.
 #' @param options A named list, to override some ARPACK options. See
-#'   [arpack()] for details.
+#'   [arpack()] for details.In packages, document ... with this standard tag:
```

Comment:
Is "In packages, document ... with this standard tag:" intentional here?

Reply:
oops no, I'll fix it, thank you